practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,38 @@
+.container-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 61px 0;
+}
+
+.header-logo {
+  width: 180px;
+  height: 52px;
+
+  & .logo-link { display: block; }
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-list {
+  display: flex;
+  list-style-type: none;
```

Comment:
Please help me check my comment again 

```You have used list-style: none for ul tag in reset file. So you don't need to override it here```

Reply:
I've removed style for ul in `reset.css` because the webpage has a single <ul> so I have to put style in `header.css`.